README.md

```diff
@@ -74,3 +72,17 @@ C:\devops\DevOps-Course-Starter>poetry run flask run
 
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+Testing 
+The application NEEDS a trello board called ToDo case sensitive 
+
+End to End Tests
+
+To run the end to end tests you will need 
+
+1. To ensure the GeckoDrive.exe is downloaded and placed in the root of your project 
+2. To Install the Firefox Browers as this is the driver that the code is referencing 
+3. The e2e tests create a new board called E2E test Board, which will be destroyed at the end of the test. 
+
```

Comment:
It would be nice to also provide the user with the commands with which they can run these, e.g. because of the static class for board ids, it's worth spelling out for the user that they'll need to run `poetry run pytest todo_app/tests` and `poetry run pytest todo_app/tests_e2e` separately